perfectNumber/PerfectNumberProgram.java

```diff
@@ -0,0 +1,23 @@
+package perfectNumber;
+
+public class PerfectNumberProgram {
+    public static long isPerfect(long number) {
+        long sumOfNumber = 0;
+        for (int index = 1; index <= number / 2; index++) {
+            if (number % index == 0) {
+                sumOfNumber = sumOfNumber + index;
+            }
+        }
+        return sumOfNumber;
+    }
+
+    //Write a program for Number is a Perfect Number or not
+    public static void main(String args[]) {
```

Comment:
Please add problem statement in proper place.

Reply:
Done